From 901c0b2dbe5becd326ddfe7d3081954cc7a13f70 Mon Sep 17 00:00:00 2001
From: Andrew Wang <andrew.wang@cloudera.com>
Date: Thu, 2 May 2013 14:33:59 -0700
Subject: [PATCH 057/596] MR1: MAPREDUCE-5193. A few MR tests use block sizes
 which are smaller than the default minimum block
 size. Contributed by Andrew Wang.

Reason: Broken test
Author: Andrew Wang
Ref: CDH-11911
(cherry picked from commit 9d53c3d05e153ba4be63f55e0aaf9d8a74559acf)
(cherry picked from commit 4c514a3903374f3f92ce17f819ebd6317b24a7b1)
(cherry picked from commit 6f7970cd53e50b98f1505e896694072ac503e24b)
(cherry picked from commit 91f3af3b9ad334c981969a1bebd7cdf129abfd7b)
---
 hadoop-mapreduce1-project/src/test/hdfs-site.xml |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hadoop-mapreduce1-project/src/test/hdfs-site.xml b/hadoop-mapreduce1-project/src/test/hdfs-site.xml
index 1032c9a..b311e2f 100644
--- a/hadoop-mapreduce1-project/src/test/hdfs-site.xml
+++ b/hadoop-mapreduce1-project/src/test/hdfs-site.xml
@@ -9,5 +9,10 @@
     <name>hadoop.security.authentication</name>
     <value>simple</value>
   </property>
+  <!-- Disable min block size since most tests use tiny blocks -->
+  <property>
+      <name>dfs.namenode.fs-limits.min-block-size</name>
+      <value>0</value>
+  </property>
 
 </configuration>
-- 
1.7.9.5

